src/extension/app/components/plugin/plugin-action-bar.js

```diff
@@ -247,6 +252,20 @@ export class PluginActionBar extends ConnectedElement {
       </sp-action-group>`;
   }
 
+  async handleItemSelection(event) {
+    const { value } = event.target;
+
+    const menu = await this.sidekickMenu;
+    menu.removeAttribute('open');
+
+    if (value === 'open-help') {
+      this.appStore.openPage('https://www.aem.live/docs/sidekick');
+      return;
+    }
+
+    this.appStore.fireEvent(value);
```

Comment:
I don't think we need to fire events (for custom code to listen to), we just need `sampleRUM` for usage tracking.

Reply:
This is how we notify the content script to close the extension or add/remove the project. Was pretty sure that's how I saw this current sidekick working. Will double check.

Reply:
Here is what I was referring to.

https://github.com/adobe/helix-sidekick-extension/blob/a691e99c085c06227304b41ac7729f6b0ef66842/src/extension/sidekick.js#L82

Reply:
I think the `hidden` event is useful to have in the API because custom cod might want to react to it. Not sure if `add-project` or `remove-project` are on the same level of usefulness. We could keep those internal and just fire RUM calls.